app/assets/javascripts/edit_plan.js

```diff
@@ -73,6 +73,7 @@ $(document).on("ready ajax:success", function() {
 
     $('#applied_pct_1').attr('step',0.01)
     var total = parseFloat($('#aptc_applied_total').val().replace(/\$/, ""));
+    var max_aptc = document.getElementById("max_tax_credit");
```

Comment:
keep this consistent with other variables. this should be something like below -

```
var max_aptc = parseFloat(document.getElementById("max_tax_credit").innerHTML);
```